prod/westeurope/internal/appbackend/app_service/terragrunt.hcl

```diff
@@ -221,6 +224,7 @@ inputs = {
 
       // BPD
       ALLOW_BPD_IP_SOURCE_RANGE : "appbackend-ALLOW-BPD-IP-SOURCE-RANGE"
+      JWT_SUPPORT_TOKEN_PRIVATE_RSA_KEY : "appbackend-JWT-SUPPORT-TOKEN-PRIVATE-RSA-KEY"
```

Comment:
Is this required for BPD only? If so why not to use the same naming convention used above? I mean with BPD somewhere?

Reply:
I'd rather keep this naming because, this token's certificate could be used also in other context, not only for BPD.